flutter/lib/src/default_integrations.dart

```diff
@@ -221,3 +224,47 @@ class LoadAndroidImageListIntegration extends Integration {
     options.sdk.addIntegration('loadAndroidImageListIntegration');
   }
 }
+
+/// a PackageInfo wrapper to make it testable
+typedef PackageLoader = Future<PackageInfo> Function();
+
+/// an Integration that loads the Release version from Native Apps
+/// or SENTRY_RELEASE and SENTRY_DIST variables
+class LoadReleaseIntegration extends Integration<SentryFlutterOptions> {
```

Comment:
not so happy with this being an `Integration` but here is the thing, this should be the last thing to be executed because we want all the previous integrations to be run (so uncaught exception hooks from Flutter/Isolate, etc...) are in place.